dev/io.openliberty.microprofile.telemetry.1.1.internal/original.bnd

```diff
@@ -16,7 +16,8 @@ Bundle-SymbolicName: io.openliberty.microprofile.telemetry.1.1.internal
 
 Export-Package: \
 	io.openliberty.microprofile.telemetry11.internal.cdi;version='1.0',\
-	io.openliberty.microprofile.telemetry11.internal.rest;version='1.0';thread-context=true
+	io.openliberty.microprofile.telemetry11.internal.rest;version='1.0';thread-context=true,\
+	io.openliberty.microprofile.telemetry11.internal.rest.javax;version='1.0';thread-context=true
```

Comment:
I suspect this does not need to be exported and does not need thread-context=true. Please try moving it to `Private-Package` below.